rd-kt/rd-framework/src/commonMain/kotlin/com/jetbrains/rd/framework/base/RdExtBase.kt

```diff
@@ -175,8 +177,15 @@ class ExtWire : IWire {
         require(newContexts === realWire.contexts) { "Can't replace ProtocolContexts on ExtWire" }
     }
 
+    @Suppress("ArrayInDataClass")
     data class QueueItem(val id: RdId, val msgSize: Int, val payoad: ByteArray, val context: List<Pair<RdContext<Any>, Any?>>)
     override val connected: Property<Boolean> = Property(false)
+    override val heartbeatAlive = connected
+
+    @UseExperimental(ExperimentalTime::class)
+    override var heartbeatInterval: Duration
+        get() = Duration.INFINITE
+        set(_) {}
```

Comment:
Silently ignoring the set value seems questionable - it might be better to throw NotSupported

Reply:
I have been thinking and realized that there is a scenario when user sets this property via IWire before connection is establsished. As well as it advises on heartbeatAlive.